src/components/RoomConnect.tsx

```diff
@@ -1,14 +1,38 @@
 import React, { FC } from 'react';
+import { Room, UserDisplay } from '../constants/Types';
+import CustomButton from './CustomButton';
 import './RoomConnect.css';
 
 type RoomConnectProps = {
+    currentRoom: Room;
+    displayName: UserDisplay;
     sendRequests: () => void;
 };
 
 // TODO: Style
 const MessageContainer: FC<RoomConnectProps> = (props: RoomConnectProps) => (
-    <div className="room-awaiting-container">
-        <button onClick={props.sendRequests}>Connect</button>
+    <div className="room-connecting-container">
+        {Object.keys(props.currentRoom.invited).map((userid, i) => {
+            if (userid === props.displayName.userid) {
+                return;
```

Comment:
Don't have to change this now, but I think we should avoid returning null when using `.map`. It'll create an empty entry in the new array, which is I guess okay because reactDOM won't render anything if its null, but probably not best to leave a null entry we don't want.

Reply:
I think i got a warning from before on this.

Reply:
Would changing to something like the following change anything? Isn't it still returning null when `userid === props.displayName.userid`?

`if (userid !== props.displayName.userid) { ... }`

Reply:
Maybe use filter instead of map?

Reply:
I don't think filter by itself would work? You would have to do filter then map, which at that point might as well just leave as map.